.buildkite/pipeline-release-redhat.yml

```diff
@@ -1,11 +1,8 @@
 
 agents:
-  image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:5827bee6
+  image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:abaeba8c
```

Comment:
I'm not sure about what is backported into `2.8`, there is no easy way to get a diff between `5827bee6` and `abaeba8c` 😕

Reply:
> I recognize that it is not practical.

Yes, that was my point. It's not obvious from looking at this PR what changes are "hidden" in the image.